libs/designsystem/src/lib/components/toast/config/toast-config.ts

```diff
@@ -5,4 +5,7 @@ export interface ToastConfig {
   animated?: boolean;
 }
 
+/**
+ * @deprecated 'danger' is deprecated. Use alert for critical events
+ */
```

Comment:
You shouldn't deprecate entire `MessageType` type - only the 'warning' option is deprecated